test/resources/mnist/distributed_customer_script_with_env_vars.py

```diff
@@ -162,8 +162,13 @@ def _preprocess_mnist(raw, withlabel, ndim, scale, image_dtype, label_dtype, rgb
 
     trainer.run()
 
+    # TODO: the MPI command makes the SM_HOSTS variable an invalid JSON
```

Comment:
This is not really actionable. It's just a statement.

Reply:
will fix